### DISCUSSION TITLE
how to configure colors with configuration file

### DISCUSSION THREAD

**Discussion by brahim-ouali** [borntyping/python-colorlog#122] [Support] [answered]
Hello I tried something like that:

```ini
[formatter_long]
format=%(log_color)s%(levelname)s:%(name)s:%(message)s
datefmt=
class=colorlog.ColoredFormatter
log_colors={'DEBUG': 'blue', 'INFO': 'green', 'WARNING': 'yellow', 'ERROR': 'red', 'CRITICAL': 'bold_red'}
```
but it's not working, how can I do to configure colors in the ini file?

Thank you

**Comment by borntyping** [comment, marked as answer]
This is not possible using the ini file approach – Python's ConfigParser format does not support dictionaries as values.

https://docs.python.org/3/library/configparser.html#supported-ini-file-structure
